test/function/samples/dynamic-import-expression/_config.js

```diff
@@ -0,0 +1,33 @@
+var assert = require( 'assert' );
+
+module.exports = {
+	description: 'Dynamic import expression replacement',
+	options: {
+		acorn: {
+			plugins: { dynamicImport: true }
```

Comment:
I tried setting these flags to `false` in all tests but that did not have any effect. Are they actually required?

Reply:
They are definitely required and will cause an error in acorn otherwise. I've definitely seen the error in my tests. let me know if you are still getting this though.